package.json

```diff
@@ -18,25 +18,26 @@
     "test": "mocha"
   },
   "dependencies": {
-    "@babel/core": "~7.15.5",
-    "@babel/eslint-parser": "~7.15.7",
+    "@babel/core": "~7.16.0",
+    "@babel/eslint-parser": "~7.16.0",
     "eslint-restricted-globals": "~0.2.0"
   },
   "devDependencies": {
-    "@lwc/eslint-plugin-lwc": "^1.1.0",
+    "@lwc/eslint-plugin-lwc": "^1.1.1",
     "@salesforce/eslint-plugin-lightning": "^0.1.0",
-    "eslint": "^7.26.0",
-    "eslint-plugin-import": "^2.23.2",
-    "eslint-plugin-jest": "^24.5.2",
-    "husky": "^7.0.0",
-    "lint-staged": "^11.2.0",
-    "mocha": "^9.1.2",
-    "prettier": "^2.4.1"
+    "eslint": "^8.2.0",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jest": "^25.2.4",
+    "husky": "^7.0.4",
+    "lint-staged": "^11.2.6",
+    "mocha": "^9.1.3",
+    "prettier": "^2.4.1",
+    "semver": "^7.3.5"
   },
   "peerDependencies": {
     "@lwc/eslint-plugin-lwc": "^1.0.0",
     "@salesforce/eslint-plugin-lightning": "^0.1.0",
```

Comment:
`@salesforce/eslint-plugin-lightning` still doesn't support eslint@8.

Reply:
@nolanlawson The PR is ready for review: https://github.com/salesforce/eslint-plugin-lightning/pull/9

Reply:
I updated the PR with the new `@salesforce/eslint-plugin-lightning` version.